framework/tests/viewpointtest/VarargType.java

```diff
@@ -0,0 +1,27 @@
+import viewpointtest.quals.*;
+
+public class VarargType {
```

Comment:
What is the `Type` in the class name referring to?

Reply:
changed it to VarargsConstructor.